crypto/store/store_lib.c

```diff
@@ -362,7 +367,7 @@ OSSL_STORE_INFO *OSSL_STORE_load(OSSL_STORE_CTX *ctx)
 
     ctx->loading = 1;
  again:
-    if (OSSL_STORE_eof(ctx))
+    if (OSSL_STORE_eof(ctx) == 1)
```

Comment:
Why this change? The OSSL_STORE_eof is supposed to return 0 or 1.

Reply:
See below.

Reply:
Reverted